admin/starter/.gitattributes

```diff
@@ -0,0 +1,2 @@
+.gitattributes export-ignore
+.github/ export-ignore
```

Comment:
`/` prefix need to be added to ensure only ignore root dir/file, see https://github.com/codeigniter4/CodeIgniter4/blob/develop/admin/framework/.gitattributes